apps/desktop/src/app/sync/protobuf.ts

```diff
@@ -21,6 +21,21 @@ export const NucliaProtobufConverter = (buffer: Uint8Array) =>
     }
   });
 
+const BASE64_MARKER = ';base64,';
+export const convertDataURIToBinary = (dataURI: string) => {
+  const base64MarkerIndex = dataURI.indexOf(BASE64_MARKER);
+  const base64Index = base64MarkerIndex === -1 ? 0 : dataURI.indexOf(BASE64_MARKER) + BASE64_MARKER.length;
+  const base64 = dataURI.substring(base64Index);
```

Comment:
that's really complex and difficult to understand
what about:
```ts
const base64 = dataURI.includes(BASE64_MARKER) ? dataURI.split(dataURI)[1] : dataURI;
```
?

Reply:
way better! thanks